ui/pages/swaps/loading-swaps-quotes/__snapshots__/loading-swaps-quotes-stories-metadata.test.js.snap

```diff
@@ -27,4 +27,4 @@ exports[`storiesMetadata matches expected values for storiesMetadata 1`] = `
   },
 }
-`;
\ No newline at end of file
+`;
```

Comment:
This change seems unintentional, and should likely be removed from the PR

Reply:
I've updated snapshots and this is what I got. When I run `yarn jest --updateSnapshot` now, it doesn't give me anything new.